src/widgets/hierarchical-menu/hierarchical-menu.js

```diff
@@ -53,6 +57,8 @@ function hierarchicalMenu({
     container,
     attributes,
     separator = ' > ',
+    rootPath = null,
```

Comment:
Well it seems that setting null by default is not in fashion anymore. Could you change to no default value?